api/src/unraid-api/config/config.registry.ts

```
@@ -0,0 +1,30 @@
+        this.configTypes.set(configName, configType);
+    }
+
+    static getConfigType(configName: string) {
+        return this.configTypes.get(configName);
+    }
+
+    static getConfigToken(configName: string) {
+        const configType = ConfigRegistry.getConfigType(configName) ?? '';
+        return makeConfigToken(configName, configType);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)